configs/desktop_savant_events_schemas.json

```diff
@@ -0,0 +1,1122 @@
+{
+  "source": "telemetry-cohorts",
+  "filters": {
+    "docType": [
+      "main"
+    ],
+    "appName": [
+      "Firefox"
+    ],
```

Comment:
appName is not a partition field for "telemetry-cohorts", but I assume that's fine; this will just get treated as a ping-level filter rather than a partitioning filter.

Reply:
IIRC the experiment was release-only -- should we also filter on that?